package.json

```diff
@@ -50,5 +52,44 @@
     "bugs": {
         "url": "https://github.com/Microsoft/axe-sarif-converter/issues"
     },
-    "homepage": "https://github.com/Microsoft/axe-sarif-converter#readme"
+    "homepage": "https://github.com/Microsoft/axe-sarif-converter#readme",
+    "license-check-and-add-config": {
+        "folder": "./",
+        "license": "copyright-header.txt",
+        "exact_paths_method": "EXCLUDE",
+        "exact_paths": [
+            "./.vscode",
+            "./.git",
+            "./.github",
+            "./node_modules",
+            "./copyright-header.txt"
+        ],
+        "file_type_method": "INCLUDE",
+        "file_types": [
+            ".ts",
+            ".d.ts",
+            ".js",
+            ".yaml",
+            ".md",
+            ".txt",
+            ".xml"
```

Comment:
Does this do anything, given that we don't specify a license_formats entry for it?

Reply:
license-check-and-add has default format for most of the file types. We specify entry point when we hope to customize.